acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNodeConfigBuilder.java

```diff
@@ -317,6 +317,18 @@ public TekuNodeConfigBuilder withWritableKeystorePath(
     return this;
   }
 
+  public TekuNodeConfigBuilder withDefaultLogging() {
+    LOG.debug("Removed log-destination from config");
+    configMap.remove("log-destination");
+    return this;
+  }
+
+  public TekuNodeConfigBuilder withInvalidKeystorePath(final String path) {
```

Comment:
This method should be: `withKeystorePath(final String path)` and the user decides if they put a invalid path or not.